src/content/docs/logs/get-started/live-archives.mdx

```diff
@@ -108,7 +86,9 @@ A partition is a way to group logs based on specific criteria. Creating partitio
    Go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Logs > Data partitions**</DoNotTranslate> to create a partition using live archives.
 </figcaption>
 
-You can also edit an existing partition clicking the <Icon name="fe-more-horizontal"/> icon on the <DoNotTranslate>**Data partitions**</DoNotTranslate> page.
+5. The <DoNotTranslate>**Total effective retention**</DoNotTranslate> days will be updated to reflect the regular retention _and_ the [live archives retention](/docs/logs/get-started/live-archives-billing/#what-counts).
+
+You can also edit an existing partition clicking the <Icon name="fe-more-horizontal"/> icon on the <DoNotTranslate>**Data partitions**</DoNotTranslate> page. If you use an existing partition, the same parameters that you selected for the original partition will apply to your live archive data. 
```

Comment:
@homelessbirds how do you get to this partitions page without dropping out of the current live archives screen flow?

I like the next sentence about the parameters for the existing partitions.